### PR TITLE
Improve docs

### LINKED ISSUE #713: Update README to include section about letsencrypt and nginx proxy
Our preferred way is to always use HTTPS and behind nginx proxy wit updated cipher settings with rate limiting to prevent brute force.

We should have a section in readme to help people with how to configure nginx and weechat for this, and then we can also change our default port to 443.

### PR DESCRIPTION
**Pull request by lorenzhs**
Let's Encrypt this shit!
Fixes #713, #788

Related: #759

### CODE REVIEW

**Comment by cormier** [comment]
The documentation is good, However the alert shows up on https://latest.glowing-bear.org/pull/853/ even though we're on https?

**Comment by lorenzhs** [comment]
hum crap I didn't test that. Looks like the ng-if doesn't work there. Hm.

File: index.html [hunk 1]
```diff
@@ -125,23 +126,43 @@ <h4 class="panel-title">
...
-              <div>To start using glowing bear, please enable the relay plugin in your WeeChat client:
+              <p><span class="label label-danger">WeeChat version 0.4.2 or higher is required—we recommend at least 1.0.</p>
+              <p>To start using Glowing Bear, follow the instructions below to set up an encrypted relay. All communication goes directly between your browser and your WeeChat relay! This means that your server must be accessible. We never see any of your data or your password, and you don't need to trust a "cloud". All settings, including your password, are saved locally in your own browser between sessions.</p>
+              <div class="alert alert-warning" ng-if="window.location.protocol !== 'https:'"><strong>You're using Glowing Bear over an unencrypted connection (http://). This is not recommended!</strong> We recommend using our secure hosted version at <a href="https://www.glowing-bear.org/">https://www.glowing-bear.org/</a>, or <a href="https://latest.glowing-bear.org/">https://latest.glowing-bear.org</a> for the latest and greatest development version. You can still follow the instructions below to set up an encrypted relay, though.</div>
```
Comment (lorenzhs) [inline index.html]: This one doesn't work either. Hmpf.

Comment (lorenzhs) [inline index.html]: Anyone have any ideas how this can be fixed (or why it doesn't work)?

**Comment by lorenzhs** [comment]
Oh, didn't think of electron. You're absolutely right.

**Pull request merged**